src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -77,6 +74,9 @@ public static void Main()
 ", options: TestOptions.DebugExe);
             var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
             comp.TestOnlyCompilationData = accessorBindingData;
+
+            Assert.Empty(comp.GetTypeByMetadataName("C").GetMembers(GeneratedNames.MakeBackingFieldName("P")));
```

Comment:
>GetMembers(GeneratedNames.MakeBackingFieldName("P")) [](http://example.com/codeflow?start=57&length=52)

Could we simplify the check to assert that there are no fields? The current check depends on us guessing the "right" name, but that is just another thing to get wrong and fail to detect a wrong behavior.

Reply:
It looks like this suggestion is applicable to other similar checks.